cpp/models/secir/secir_parameters_io.cpp

```diff
@@ -196,86 +134,82 @@ namespace details
+                if (date_df == offset_date_by_days(date, 0)) {
+                    num_inf[age] += (1 - mu_C_R[age]) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                    num_rec[age] += region_entry.num_confirmed;
+                }
+                if (date_df == offset_date_by_days(date, days_surplus)) {
+                    num_car[age] +=
+                        (2 * mu_C_R[age] - 1) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // -R9
+                if (date_df == offset_date_by_days(date, -t_car_to_rec[age] + days_surplus)) {
+                    num_car[age] -= mu_C_R[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // +R2
+                if (date_df == offset_date_by_days(date, +t_exp_to_car[age] + days_surplus)) {
+                    num_exp[age] += mu_C_R[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // +R3
+                if (date_df == offset_date_by_days(date, +t_car_to_inf[age] + days_surplus)) {
+                    num_car[age] += (1 - mu_C_R[age]) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                    num_exp[age] -= (1 - mu_C_R[age]) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // R2 - R9
+                if (date_df == offset_date_by_days(date, t_exp_to_car[age] - t_car_to_rec[age] + days_surplus)) {
+                    num_exp[age] -= mu_C_R[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // R2 + R3
+                if (date_df == offset_date_by_days(date, t_exp_to_car[age] + t_car_to_inf[age] + days_surplus)) {
+                    num_exp[age] += (1 - mu_C_R[age]) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // -R4
+                if (date_df == offset_date_by_days(date, -t_inf_to_rec[age])) {
+                    num_inf[age] -= (1 - mu_C_R[age]) * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // -R6
+                if (date_df == offset_date_by_days(date, -t_inf_to_hosp[age])) {
+                    num_inf[age] -= mu_I_H[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                    num_hosp[age] += mu_I_H[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                }
+                // -R6 - R7
+                if (date_df == offset_date_by_days(date, -t_inf_to_hosp[age] - t_hosp_to_icu[age])) {
+                    num_inf[age] +=
+                        mu_I_H[age] * mu_H_U[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                    num_hosp[age] -=
+                        mu_I_H[age] * mu_H_U[age] * scaling_factor_inf[age] * region_entry.num_confirmed;
+                    if (read_icu) {
+                        num_icu[age] +=
```

Comment:
I think that would be a bad comment since it just states the line below again. and I think it's not even correct in this line, I think you mixed up the m_X_Y variables.